R/propensity-weighting.R

```diff
@@ -0,0 +1,30 @@
+#' Calculates inverse probability of treatment weights using a random forest
+#'
+#' @param df <`tbl`> Data frame on which to perform weighting
+#' @param treatment <`character(1)`> String with name of treatment column
+#' @param ivs <`character`> Character vector with names of independent variabes to use in propensity score model
+#'
+#' @return <`tbl`> Data frame with new column for propensity weights
+#'
+#' @import purrr
+#'
+#' @export
+propensity_weighting <- function(df, treatment, ivs) {
```

Comment:
What about naming the function `add_propensity_weights()` to make it more of a verb?

Reply:
Agreed. Should we consider more verb-y names for our other functions? `preprocess` is probably fine, but `descriptives` is a noun

Reply:
`descriptives` could become `describe`?

Reply:
Maybe, we should think about it